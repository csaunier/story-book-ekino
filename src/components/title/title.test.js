import React from 'react'

import { shallow } from 'enzyme'

import Title from './title'

describe('component Title', () => {
  it('should render', () => {
    const component = shallow(<Title text="text" />)

    expect(component).toMatchSnapshot()
  })

  it('should render with a subtitle', () => {
    const component = shallow(<Title text="text" subtitle="subtitle" />)

    expect(component).toMatchSnapshot()
  })
})
