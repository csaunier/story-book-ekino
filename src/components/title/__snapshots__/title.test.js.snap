// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component Title should render 1`] = `
<div
  className="wrapper"
>
  <h1
    className="title"
  >
    text
  </h1>
</div>
`;
